### PR TITLE
started on controller for server

### PR DESCRIPTION
**Pull request by briansai**

### CODE REVIEW

**Comment by briansai** [comment]
@lighthurst  Hey Aaron.  Can you please review the controller.js in server please?   Thanks!

File: server/controller/controller.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+const bodyParser = require('body-parser');
+const path = require('path');
+const model = require('../model/model.js');
+
+const app = express();
+const PORT = 3000;
+
+
+app.use(bodyParser.json());
+app.listen(PORT);
```
Comment (lighthurst) [inline server/controller/controller.js]: Add the listen at the bottom of the page.

File: server/controller/controller.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+console.log('Listening to ', PORT);
```
Comment (lighthurst) [inline server/controller/controller.js]: Asynchronous code: Pass in as a second argument to app.listen an anonymous function that includes the console.log invocation in its body. Like app.listen(PORT, () => console.log....);

File: server/controller/controller.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+app.get('/description', (req, res) => {
+  const params = req.query;
+
+  model.get(params, (err, result) => {
+    if (err) {
+      res.status(404).send(err);
+    }
+
+    res.status(200).send(JSON.parse(result.body));
```
Comment (lighthurst) [inline server/controller/controller.js]: Perhaps refactor to use res.json(result.body).

File: server/controller/controller.js [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+
+
+app.use(bodyParser.json());
+app.listen(PORT);
+console.log('Listening to ', PORT);
+
+app.use(express.static(path.join(__dirname, '../client')));
+
+app.get('/description', (req, res) => {
+  const params = req.query;
```
Comment (lighthurst) [inline server/controller/controller.js:15]: What data is contained in req.query? Is it only the roomId?

**Review by lighthurst — CHANGES_REQUESTED** [review]
Nice work. A few comments to change.

File: server/controller/controller.js (hunk 4 above)
Comment (briansai) [reply server/controller/controller.js:15]: Yes, it's only the room id.

Comment (briansai) [reply server/controller/controller.js:15]: Should I use the variable method like '/descriptions/:id' ?

**Review by lighthurst — APPROVED** [review]

**Pull request merged**